examples/basic_file.py

```diff
@@ -14,7 +14,7 @@
 
 print("--------------------------------------------------------------------------------")
 print("Scenario: single file with prompt injection detection")
-file="./test_data/pi-test.txt"
+file="./examples/test_data/pi-test.txt"
```

Comment:
Let's be a bit smarter here and let's make the file references relative to the file itself. It's not really good practice when there are expectations on the working directory.

Reply:
hmm, not sure why its showing as part of my PR. This were Satyam's changes, I thought it was merged maybe he added it here. 

Will me take care of it.